tests/ImageSharp.Tests/Formats/Bmp/BmpDecoderTests.cs

```diff
@@ -611,6 +611,11 @@ public void BmpDecoder_CanDecode_Os2v2Header<TPixel>(TestImageProvider<TPixel> p
         public void BmpDecoder_CanDecode_Os2BitmapArray<TPixel>(TestImageProvider<TPixel> provider)
             where TPixel : unmanaged, IPixel<TPixel>
         {
+            if (TestEnvironment.IsLinux)
+            {
+                throw new SkipTestException("See discussion on https://github.com/SixLabors/ImageSharp/pull/2890");
+            }
```

Comment:
The Linux failure is a mystery: it only shows when running the test suite. If I run a console app against the compiled `ImageSharp.dll` on Ubuntu, the image loads without an issue. I have spent an hour trying to hook a debugger into VS Code, but it keeps ignoring my breakpoints.

I ended up skipping the test on Linux. (1) It's likely a false alert that appears only with `dotnet test` (2) likely not a regression against the previous version, (3) an edge-case anyways.

Reply:
I think it is ok to skip those bmp tests on linux, they are extremely niche bitmap variants. It's also a mystery to me why this is different on linux than on windows.

Reply:
It was an LFS issue (╯°□°)╯︵ ┻━┻

Reply:
Wow, good find